### PR TITLE
Periodically fetch queue status

### PR DESCRIPTION
**Pull request by migerh**
For now we will poll the backend for queue updates every 3s. Since we
currently don't have an API do fetch the place id from the backend, we
provide it via an environment variable that we inject into the frontend.
Start the frontend application with

```sh
REACT_APP_PLACE_ID=<place id as reported by backend> npm start
```

### CODE REVIEW

**Comment by migerh** [comment]
The corresponding backend API does not yet exist, @Gronner is working on it. Once that is done, this change will (hopefully) work without producing a lot of 405 errors in the backend log.

File: frontend/src/state/backend.ts [hunk 1]
```diff
@@ -1,36 +1,71 @@
-import { all, call, takeEvery } from 'redux-saga/effects'
+import { all, call, takeEvery, put, delay } from 'redux-saga/effects'
+import { updateQueueCreator } from './queue';
 
 const CREATE_QUEUE = "@backend/CREATE_QUEUE";
-
-export const createQueueCreator = (placeId: string, name: string) => ({
+interface CreateQueueAction {
+  type: "@backend/CREATE_QUEUE";
```
Comment (schaeferto) [inline frontend/src/state/backend.ts]:
```suggestion
  type: CREATE_QUEUE;
```

File: frontend/src/state/backend.ts [hunk 2]
```diff
@@ -1,36 +1,71 @@
...
+export const createQueueCreator = (placeId: string, name: string): CreateQueueAction => ({
   type: CREATE_QUEUE,
   placeId,
   name
 });
 
-function* queueSaga(action: any) {
+const FETCH_QUEUES = "@backend/FETCH_QUEUES";
+interface FetchQueuesAction {
+  type: "@backend/FETCH_QUEUES";
```
Comment (schaeferto) [inline frontend/src/state/backend.ts]:
```suggestion
  type: FETCH_QUEUES;
```

File: frontend/src/state/backend.ts [hunk 3]
```diff
@@ -1,36 +1,71 @@
...
+    } else {
+      console.error("Queue fetch failed: ", response);
     }
   }
 }
 
+function* watchBackend() {
+  // wait 3 seconds
+  while (true) {
+    yield put(fetchQueuesCreator(process.env.REACT_APP_PLACE_ID || "someplace"));
```
Comment (schaeferto) [inline frontend/src/state/backend.ts]: Hier könnte dann der Wert aus der Url genommen werden siehe #47

File: frontend/src/state/backend.ts [hunk 4]
```diff
@@ -1,36 +1,71 @@
...
+      },
+      body: JSON.stringify({ queueName: action.name })
+    });
+    if (response.ok) {
+      console.log("Queue created, response: ", response);
+    } else {
+      console.error("Queue creation failed: ", response);
+    }
+  } else if (action.type === FETCH_QUEUES) {
+    const response = yield call(fetch, `/places/${action.placeId}/queues`, {
```
Comment (Gronner) [inline frontend/src/state/backend.ts]: Without having any idea of TypeScript, shouldn't the query option of .../queues?personDetails=short or full be somewhere here?

File: frontend/src/state/backend.ts (hunk 3 above)
Comment (migerh) [reply frontend/src/state/backend.ts]: Let's do this later.

File: frontend/src/state/backend.ts (hunk 4 above)
Comment (migerh) [reply frontend/src/state/backend.ts]: Yeah, let's fix that tomorrow.

Comment (migerh) [reply frontend/src/state/backend.ts]: full is not yet implemented in the backend. will use short for now.

**Pull request merged**